mne/chpi.py

```diff
@@ -317,6 +317,14 @@ def _setup_chpi_fits(info, t_window, t_step_min, method='forward',
 
     # Set up magnetic dipole fits
     picks_meg = pick_types(info, meg=True, eeg=False, exclude=exclude)
+    if exclude == 'bads':
+        logger.debug('Static bad channels (%d): %s'
+                     % (info['bads'].__len__(), ' '.join(map('{:s}'.format,
+                                                             info['bads']))))
+    elif exclude is not None:
```

Comment:
you should just be able to do `else` here because `exclude` should never be `None` (it's not an allowed argument)